modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc

```diff
@@ -34,9 +36,9 @@ NOTE: Although you can use any document attribute that clearly indicates the var
 = Module B Heading
 ----
 
- . In the assembly file, define the `+++:context:+++` attribute immediately above any included modules that are being reused. How you define the attribute depends on whether the module is included once in multiple assemblies, or is included multiple times in a single assembly.
+ . In the assembly file or the master book file, define the `+++:context:+++` attribute immediately above any included modules that are being reused. How you define the attribute depends on whether the module is included once in multiple assemblies, or is included multiple times in a single assembly.
```

Comment:
This is IMO important. If `{context}` is not set in the `master.adoc` file, we will get weird anchor IDs of modules included until the first assembly that reuses modules appears.